server/emails/email.js

```diff
@@ -0,0 +1,29 @@
+import sgMail from "@sendgrid/mail";
```

Comment:
This is a repetition on @teepha  logic, Please refactor an use methods so we keep our code DRY

Reply:
Please refactor this logic it is too tightly coupled and does not leave room for expansion